docs/source/deployment/prefect.md

```diff
@@ -113,8 +92,124 @@ def build_and_register_flow(pipeline_name, env):
     flow.run_agent()
 
 
+class KedroInitTask(Task):
+    """Task to initialize kedro session"""
+
+    def __init__(
+        self,
+        pipeline_name: str,
+        package_name: str = None,
```

Comment:
This shouldn't really be `None` though right? It wouldn't fail, but you'd get an odd name of "None_initialization" in the snippet below `f"{package_name}_initialization"`. I think you can just add it as a required argument.

Reply:
:+1: